playground/src/CodeEditor.tsx

```diff
@@ -2,21 +2,32 @@ import * as React from 'react';
 import * as monacoEditor from 'monaco-editor';
 import { FlexColDiv } from './FlexDivs';
 
-export interface ICommentSyntaxMarker {
-  message: string;
-
+export interface ISyntaxLocation {
   /**
-   * Beginning
+   * Beginning position as a character index of the text in the editor
    */
   pos: number;
 
   /**
-   * End
+   * End position as a character index of the text in the editor
    */
   end: number;
 }
 
-export interface IMonacoWrapperProps {
+export interface ISyntaxMarker extends ISyntaxLocation {
```

Comment:
>ISyntaxMarker [](start = 17, length = 13)

Could you add some comments explaining the relationship between `ISyntaxMarker`, `ISyntaxStyle`, and `ITrackedSyntaxStyle`?

Maybe also explain the difference between a "marker" and a "style" since most people will be unfamiliar with the Monaco API.